hyperopt/tests/test_mongoexp.py

```diff
@@ -286,6 +329,13 @@ def passthrough(x):
     return x
 
 
+def passthrough_with_attachments(x):
```

Comment:
could you give a better name than x and typehint it? Also could you move this function after `test_trial_attachments` ?